doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,149 +3,85 @@
+*  View coverage reports for rule-based models. The test scenarios designer displays the test coverage statistics in the *Coverage Report* panel on the right side of the test scenarios designer.
 
-For more information about DMN support in {PRODUCT} {PRODUCT_VERSION}, see the following resources:
+* Download coverage reports for rule-based and DMN-based models. You can now download the test scenario coverage reports in CSV format.
 
-* {URL_DMN_MODELS}[_{DMN_MODELS}_]
-* {URL_GETTING_STARTED_DECISION_SERVICE}[_{GETTING_STARTED_DECISION_SERVICE}_]
+* You can run test scenarios with stateless KIE sessions. To configure a stateless KIE session, expand the *Settings* panel of the test scenario and select or clear the *Stateless Session* check box to indicate whether or not the KIE session is stateless or not.
 
+NOTE: The test scenario 'scesim' file format is not public. You can only edit the 'scesim' file using the test scenario editor.
 
 == {OPENSHIFT}
 
-=== JMS capabilities for {KIE_SERVER}
-JMS capabilities for {KIE_SERVER} are now available. With JMS capabilities you can interact with the server through the JMS API using an external AMQ message broker.
-
-=== {CENTRAL} extensions for OpenShift and OpenShift Startup Strategy for {KIE_SERVER}
-
-All deployments of {PRODUCT} on
-ifdef::DM[]
-{OPENSHIFT}
-endif::DM[]
-ifdef::PAM[]
-{OPENSHIFT}, except the fixed managed template (`{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml`),
-endif::PAM[]
-now use OpenShift Startup Strategy for all {KIE_SERVERS} by default (the `KIE_SERVER_STARTUP_STRATEGY` environment variable) and also have the {CENTRAL} extension for OpenShift enabled (the `KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` environment variable). These features enable {CENTRAL}
-ifdef::PAM[]
-and {CENTRAL}
-Monitoring
-endif::PAM[]
-to discover all {KIE_SERVERS} in the same name space automatically. They eliminate the need to configure a controller.
-
-=== Full support for {OPENSHIFT} 4.x in OpenShift Operator
-
-The OpenShift Operator supplied with {PRODUCT} {PRODUCT_VERSION} fully supports OpenShift 4.x.
-
-=== {OPENSHIFT} images use Red Hat Entreprise Linux 8 and OpenJDK 11
-{PRODUCT} container images for {OPENSHIFT} now use a Red Hat Enterprise Linux 8 base layer and OpenJDK 11.
-
-== {PLANNER}
+=== High available event-driven decisioning
+A reference implementation for high available event-driven decisioning on {OPENSHIFT} is available. You can use this reference implementation to deploy your complex event processing rules in an OpenShift environment to enable processing of complex event series by the decision engine with high availability.
 
-=== Enhancement to Java API
+=== Automatic major and minor patch updates in operator deployments on {OPENSHIFT}
```

Comment:
No, it is "minor and patch version". Major versions (to 8.x) are not updated, only patch (7.5.0>7.5.1) and minor (7.5.x>7.6) versions